bindings/java/CMakeLists.txt

```diff
@@ -140,7 +140,7 @@ set_target_properties(java_workloads PROPERTIES
 target_link_libraries(java_workloads PUBLIC fdb_c ${JNI_LIBRARIES})
 target_include_directories(java_workloads PUBLIC ${JNI_INCLUDE_DIRS})
 
-set(CMAKE_JAVA_COMPILE_FLAGS "-source" "1.8" "-target" "1.8")
+set(CMAKE_JAVA_COMPILE_FLAGS "-source" "1.8" "-target" "1.8" "-XDignore.symbol.file")
```

Comment:
I'm not really familiar with this stuff, but do you know if [this suggestion](https://github.com/apple/foundationdb/issues/3002#issuecomment-618071252) also works but with a narrower scope? If so I'd prefer that

Reply:
I tried - this doesn't help. I will leave the compiler flag in and leave it to someone else to fix it.